src/app.ts

```diff
@@ -0,0 +1,42 @@
+import * as Sentry from "@sentry/node";
+import * as express from "express";
+import InternalIp from "internal-ip";
+import middleware from "./middleware";
+import { AbstractRouter } from "./routers/abstractrouter";
+import logger from "./utils/logger";
+
+export class App {
+    private port: number;
+    private routers: AbstractRouter[];
+
+    constructor(routers: AbstractRouter[], port: number) {
+        this.routers = routers;
+        this.port = port;
+
+        this.initSentry();
+    }
+
+    public start(): void {
+        
+        const server = this.initializeServer(express(), this.routers);
+        server.listen(this.port, () => {
+            logger.info(`server is running at ${InternalIp.v4()}:${this.port}`);
```

Comment:
I'm not sure if this library would introduce (new) vulnerabilities.  And as the server is listening on a specific port of the network interface card (`NIC`) and not a specific address. It would be better to print the following info:
```
logger.info(`server is running at 0.0.0.0:${this.port}`);
```
> 0.0.0.0 stands for all IP addresses on the local machine (`NIC`)